<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>维护应用平台-Version 1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/icon/maintenance_tool_tools_96px_516592_easyicon.net.ico</normaloff>:/icon/icon/maintenance_tool_tools_96px_516592_easyicon.net.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="5">
        <widget class="QToolButton" name="toolButton_search">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="icon.qrc">
           <normaloff>:/icon/icon/fake_find_magnifying_glass_search_zoom_48px_106_easyicon.net.png</normaloff>:/icon/icon/fake_find_magnifying_glass_search_zoom_48px_106_easyicon.net.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="lineEdit_description"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>故障类别：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>故障代码：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_ID"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_type"/>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>故障描述：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="6">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <column>
          <property name="text">
           <string>序号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>故障代码</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>故障名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>id</string>
          </property>
          <property name="background">
           <color>
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>故障描述</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QTextBrowser" name="textBrowser_description">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>操作指南</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0" colspan="5">
              <widget class="QTextBrowser" name="textBrowser_operation">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="frame_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>111</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <widget class="QToolButton" name="toolButton_last">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <family>新宋体</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>上一步</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QFrame" name="frame_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QRadioButton" name="radioButton_yes">
                  <property name="font">
                   <font>
                    <family>微软雅黑</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: qlineargradient(spread:pad, x1:0.244, y1:0.148, x2:0.552, y2:0.948864, stop:1 rgba(85, 156, 0, 255));</string>
                  </property>
                  <property name="text">
                   <string>正常</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton_no">
                  <property name="font">
                   <font>
                    <family>微软雅黑</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: qlineargradient(spread:pad, x1:0.244, y1:0.148, x2:0.552, y2:0.948864, stop:1 rgba(255, 85, 85, 255));</string>
                  </property>
                  <property name="text">
                   <string>不正常</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QFrame" name="frame_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="1">
                 <widget class="QToolButton" name="toolButton_next">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>新宋体</family>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>下一步</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QFrame" name="frame_6">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="1">
                 <widget class="QToolButton" name="toolButton_end">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>新宋体</family>
                    <weight>75</weight>
                    <bold>true</bold>
                    <stylestrategy>PreferDefault</stylestrategy>
                   </font>
                  </property>
                  <property name="text">
                   <string>结束</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="3">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="0" colspan="5">
              <widget class="QPlainTextEdit" name="plainTextEdit_result">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>80</height>
                </size>
               </property>
               <property name="plainText">
                <string/>
               </property>
               <property name="placeholderText">
                <string>检测结果补充说明...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_config"/>
   <addaction name="separator"/>
   <addaction name="action_download"/>
   <addaction name="action_upload"/>
   <addaction name="separator"/>
   <addaction name="action_login"/>
   <addaction name="action_exit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/exit_48px_27110_easyicon.net.png</normaloff>:/icon/icon/exit_48px_27110_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_download">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/refresh_48px_30803_easyicon.net.png</normaloff>:/icon/icon/refresh_48px_30803_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>更新故障条目</string>
   </property>
   <property name="toolTip">
    <string>更新故障条目</string>
   </property>
  </action>
  <action name="action_upload">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/upload_48px_28748_easyicon.net.png</normaloff>:/icon/icon/upload_48px_28748_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>上传诊断记录</string>
   </property>
  </action>
  <action name="action_config">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/cog_config_configuracion_configuration_edit_48px_249_easyicon.net.png</normaloff>:/icon/icon/cog_config_configuracion_configuration_edit_48px_249_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_login">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/login.png</normaloff>:/icon/icon/login.png</iconset>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
